shared/cfg/guest-os/Windows.cfg

```diff
@@ -315,22 +315,22 @@
         i386,i686:
             install_path = "C:\Program Files\JAM Software\HeavyLoad"
     driver_load:
-        driver_id_pattern = "(.*?):"
         driver_load_cmd = C:\devcon.exe enable @DRIVER_ID
         driver_unload_cmd = C:\devcon.exe disable @DRIVER_ID
+        driver_id_cmd = C:\devcon.exe find * | find "VirtIO"
```

Comment:
use devcon to find all virtio driver instead of detail info "VirtIO Balloon Driver",
then we can check the logs when it failed.